docs/c-runtime-library/locale-names-languages-and-country-region-strings.md

```diff
@@ -37,6 +37,22 @@ _wsetlocale(LC_ALL, L"de-DE");
 _wsetlocale(LC_ALL, L"LC_MONETARY=en-GB;LC_TIME=es-ES");
 ```
 
+
+## UTF-8 Support
+
+Starting in Windows 10 build 17134 (April 2018 Update), the Universal C Runtime supports using a UTF-8 code page. This means that narrow (char) strings passed to C runtime functions will expect strings in the UTF-8 encoding and properly understand them. To enable this, use "UTF-8" as the code page when using setlocale. For example, [setlocale(LC_ALL, ".utf8")](reference/setlocale-wsetlocale.md) will use the current ACP for the locale and UTF-8 for the code page.
```

Comment:
What does "properly understand them" mean? I'd punt 'narrow' and just go with 'char' for clarity.
To enable this <- what's this? Better to say, "To enable x <-- explain what you are enabling"
The acronym ACP isn't defined.